openfasoc/generators/common/simulation/simulation_run.py

```diff
@@ -15,6 +15,7 @@
 from os import path
 from common.simulation.utils import _print_progress
 import time
+import json
```

Comment:
Is this required?

Reply:
no, i was going to remove this stuff in cleanup
thanks!